### PR TITLE
The Rook, A mining ship for the SGV Atherwhisp

### PR DESCRIPTION
**Pull request by theboy6545**
## About The Pull Request
Capture

Adds a new mining ship, the SGV Rook, exclusive to the Atherwhisp,and changes the JSON to allow it.

## Why It's Good For The Game

Going from a solgov vessel to an NT one felt weird, and new stuff is always liked, right?
## Changelog
:cl:
add: Added a new ship, the SGV Rook.

/:cl:

Big thanks to Aerwhisp for telling me about all my screw ups in atmos and general help.

### CODE REVIEW

**Comment by DrSirKnight** [comment]
I'm just a player so my opinion doesn't matter, but the bottom area looks a bit stale.

**Comment by theboy6545** [comment]
> I'm just a player so my opinion doesn't matter, but the bottom area looks a bit stale.

Yeah, i couldn't really think of anything to put in there, plus it feels weird mapping over atmos pipes

**Comment by Vasily2013** [comment]
I think this looks ready for a test merge.

File: .gitignore [hunk 1]
```diff
@@ -216,3 +216,4 @@ tools/MapAtmosFixer/MapAtmosFixer/bin/*
 !/config/jukebox_music/sounds/exclude
 /config/title_music/sounds/*
 !/config/title_music/sounds/exclude
+nsv13.dme
```
Comment (Karmic-Skink) [inline .gitignore:219]: You what?

**Review by Vasily2013 — CHANGES_REQUESTED** [review]
remove the change to the gitignore and this'll get speedmerged as soon as I figure out why local testing no worie.

**Review by Vasily2013 — CHANGES_REQUESTED** [review]
This air alarm is too deep inside the wall, I'd suggest using the preset version that will put it closer to the south corner of the sprite. This appears to be an issue with all air alarms.
image

There are no air alarms in the washroom, bedroom, or the PACMAN generator room.

Additionally there is an air alarm in the bridge that is placed under a solgov logo, this should be altered. Otherwise from a bare bones functionality standpoint it looks good.

**Comment by theboy6545** [comment]
Alright, made all the requested changes.

**Comment by Vasily2013** [comment]
image
round-start active turfs. You want to use airless tiles here.

**Comment by theboy6545** [comment]
RIP travis, and the changes are done.

**Review by Vasily2013 — APPROVED** [review]
YEET

**Pull request merged**